libs/designsystem/src/lib/polyfills/intersection-observer-polyfill-loader.js

```diff
@@ -1,14 +1,14 @@
 (function () {
-    // A technique for loading polyfills only when needed. Details here:
-    // https://philipwalton.com/articles/loading-polyfills-only-when-needed/
-    if (polyfillNeeded()) {
-      // Browsers that doesn't support IntersectionObserver must load the polyfills first:
-      var script = document.createElement('script');
-      script.src = 'kirby/polyfills/intersection-observer-polyfill.min.js';
-      document.head.appendChild(script);
-    }
+  // A technique for loading polyfills only when needed. Details here:
+  // https://philipwalton.com/articles/loading-polyfills-only-when-needed/
+  if (polyfillNeeded()) {
+    // Browsers that doesn't support IntersectionObserver must load the polyfills first:
+    var script = document.createElement('script');
+    script.src = 'kirby/polyfills/intersection-observer-polyfill.min.js';
+    document.head.appendChild(script);
+  }
 
-    function polyfillNeeded() {
-        return (typeof IntersectionObserver === 'undefined');
-    }
-  })();
+  function polyfillNeeded() {
+    return typeof IntersectionObserver === 'undefined';
```

Comment:
Are we completely sure we want to remove optional parentheses?

For: Sometimes they make things easier to read and understand
Against: Clutter. Anything else?

May I suggest that we postpone decisions on which stylelint rules to use in the config? We could spend some time getting used to the current rules and the have a meeting and decide later if we want to change anything.

Reply:
This is a tslint setting, and the linter has been run when this file was staged applying prettier changes. So lets take this in another round when configuring linting.

Functionally it should make no difference.